packages/manager/apps/container/src/core/container/ContainerProvider.jsx

```diff
@@ -31,12 +37,17 @@ export const ContainerProvider = ({ children }) => {
       })
       .then((value) => {
         let version = null;
-        if (value['pnr:betaV1'] === true) {
-          version = BETA_V1;
-        }
-        if (value['pnr:betaV2'] === true) {
-          version = BETA_V2;
+        if (isBetaForced()) {
+          version = getBetaVersionFromLocalStorage();
+        } else {
+          if (value['pnr:betaV1'] === true) {
+            version = BETA_V1;
+          }
+          if (value['pnr:betaV2'] === true) {
+            version = BETA_V2;
+          }
         }
```

Comment:
***nitpick***

Also, this whole block might be better inside a function. Something like `getBetaVersion(input)` that returns a value. 

So that you would only write : 

```
const version = getBetaVersion(value);
```